pyro/infer/mcmc/logger.py

```diff
@@ -38,22 +39,72 @@
 # OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 
-# TODO: Remove once https://github.com/tqdm/tqdm/issues/650 is resolved.
-class ProgressBar(tqdm):
-    def __init__(self, *args, **kwargs):
-        super(ProgressBar, self).__init__(*args, **kwargs)
+class ProgressBar(object):
+    """
+    Initialize progress bars using :class:`~tqdm.tqdm`.
+
+    :param int warmup_steps: Number of warmup steps.
+    :param int num_samples: Number of MCMC samples.
+    :param int min_width: Minimum column width of the bar.
+    :param int max_width: Maximum column width of the bar.
+    :param int pos: Position of the bar (e.g. in the case of
+        multiple parallel samplers).
```

Comment:
nit: this `pos` parameter is not available in init method.

Reply:
Oops..sorry, it shouldn't be there. Will remove.